stories/useGravity2D.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { useGravity2D, vector as Vector } from '../src';
+import './useGravity2D.css';
+
+export default {
+  title: 'Gravity2D',
+};
+
+export const Gravity2D: React.FC = () => {
+  const [center, setCenter] = React.useState<Vector<number>>([0, 0]);
+
+  React.useLayoutEffect(() => {
+    const root = document.getElementById('root');
+    if (root) {
+      setCenter([root.clientWidth / 2, root.clientHeight / 2]);
+    }
+  }, []);
```

Comment:
This was an interesting little problem that led me to read a bit more about `useLayoutEffect`. Basically, I want to derive the center point of the `#root` node to center the `attractor`. We can't use `document.body` b/c Storybook purposefully gives it no height 🤔

<img width="970" alt="image" src="https://user-images.githubusercontent.com/19421190/71211486-143f7b00-2264-11ea-868a-eca35bea5389.png">

I haven't yet encountered a good case for `useLayoutEffect` in the wild, but this seemed like one – we basically want a sync render so the user doesn't get a flicker of the `attractor` being in the upper left corner ([0, 0]) and then suddenly changing location to the center.